### PR TITLE
Update iMac_in_the_lab.md

### PR DESCRIPTION
**Pull request by avanaudenhaege**
change password information

### CODE REVIEW

File: doc/iMac_in_the_lab.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 There are currently (update: July 2024) 4 iMacs in the lab.
-If you want information about a computer or iMac in the lab, please consult this [excel sheet](https://docs.google.com/spreadsheets/d/1U5_rjj5mzXzh_8bLA_Rx3FmDdpYIUx2Y_z8SJPnKTHg/edit?gid=0#gid=0).
+If you want information about a computer or iMac in the lab, please consult this [excel sheet](https://docs.google.com/spreadsheets/d/1U5_rjj5mzXzh_8bLA_Rx3FmDdpYIUx2Y_z8SJPnKTHg/edit?usp=sharing).
```
Comment (marcobarilari) [inline doc/iMac_in_the_lab.md:2]: Link now is not open to everyone but only to members of the gdrive

File: doc/iMac_in_the_lab.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 There are currently (update: July 2024) 4 iMacs in the lab.
-If you want information about a computer or iMac in the lab, please consult this [excel sheet](https://docs.google.com/spreadsheets/d/1U5_rjj5mzXzh_8bLA_Rx3FmDdpYIUx2Y_z8SJPnKTHg/edit?gid=0#gid=0).
+If you want information about a computer or iMac in the lab, please consult this [excel sheet](https://docs.google.com/spreadsheets/d/1U5_rjj5mzXzh_8bLA_Rx3FmDdpYIUx2Y_z8SJPnKTHg/edit?usp=sharing).
 
 It is good practice to create your own account on the computer in order not to install on the same account a different version of Matlab Tools.
 
-All iMacs have an **admin** account, password : **CPP**.
+All iMacs have an **admin** account. Ask a lab member for the password.
```
Comment (marcobarilari) [inline doc/iMac_in_the_lab.md:6]: What do you think about creating a column to the sheet file mentioned above with the pss? It won't be visible by external people. LMK what you think and I will do it.

**Review by marcobarilari — CHANGES_REQUESTED** [review]
Thank you! See my comments in the lines and let me know what you think.

File: doc/iMac_in_the_lab.md (hunk 2 above)
Comment (avanaudenhaege) [reply doc/iMac_in_the_lab.md:6]: Yes it is done. I added passwords that I know, feel free to add others.

**Comment by avanaudenhaege** [comment]
I agree with the changes. I don't know where to go now, to validate the text.

**Review by marcobarilari — APPROVED** [review]
No need to accept changes in the text here since I modified ny directly committing changes. So no need of more actions on your side. Perfect! Thank you

**Pull request merged**